specification/billing/resource-manager/Microsoft.Billing/stable/2023-04-01/billingSubscription.json

```diff
@@ -400,244 +749,605 @@
     },
-    "ValidateSubscriptionTransferEligibilityError": {
-      "description": "Error details of the transfer eligibility validation",
+    "MoveEligibilityResult": {
+      "description": "Result of the transfer eligibility validation.",
+      "type": "object",
       "properties": {
-        "code": {
-          "$ref": "#/definitions/SubscriptionTransferValidationErrorCode",
-          "description": "Error code for the product transfer validation."
+        "isMoveEligible": {
+          "description": "Specifies whether the subscription is eligible to be transferred.",
+          "type": "boolean"
         },
-        "details": {
-          "description": "Detailed error message explaining the error.",
+        "errorDetails": {
+          "description": "Error details of the transfer eligibility validation.",
+          "allOf": [
+            {
+              "$ref": "#/definitions/MoveBillingSubscriptionErrorDetails"
+            }
+          ]
+        }
+      }
+    },
+    "NextBillingCycleDetails": {
+      "description": "Billing cycle details of the product.",
+      "type": "object",
+      "properties": {
+        "billingFrequency": {
+          "description": "Billing frequency of the product under the subscription.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "RenewalTermDetails": {
+      "description": "Details for the subscription's next renewal term.",
+      "type": "object",
+      "properties": {
+        "billingFrequency": {
+          "description": "The billing frequency in ISO8601 format of product in the subscription. Example: P1M, P3M, P1Y",
           "type": "string"
         },
-        "message": {
-          "description": "The error message.",
+        "productId": {
+          "description": "Id of the product for which the subscription is purchased.",
+          "type": "string"
+        },
+        "skuId": {
+          "description": "The SKU ID of the product for which the subscription is purchased. This field is is only available  for Microsoft Customer Agreement billing accounts.",
+          "type": "string"
+        },
+        "termDuration": {
+          "description": "The duration in ISO8601 format for which you can use the subscription. Example: P1M, P3M, P1Y",
+          "type": "string"
+        },
+        "quantity": {
+          "format": "int64",
+          "description": "The number of licenses purchased for the subscription.",
+          "type": "integer"
+        },
+        "termEndDate": {
+          "description": "End date of the term in UTC time.",
           "type": "string"
         }
-      },
-      "type": "object"
+      }
     },
-    "ValidateSubscriptionTransferEligibilityResult": {
-      "description": "Result of the transfer eligibility validation.",
+    "SystemOverrides": {
+      "description": "System imposed policies that regulate behavior of the subscription.",
+      "type": "object",
       "properties": {
```

Comment:
Can we include an "overflow" property here so we can easily introduce additional system overrides in the future?

Reply:
Not sure what you mean here? Just that this should be a property bag instead of a defined object?